### PR TITLE
Changes for stateful set - V0.9.0 in IO stability degraded test. 

### PR DESCRIPTION
**Pull request by priyankaborawake**
Signed-off-by: priyanka.borawake <66418474+priyankaborawake@users.noreply.github.com>

# Problem Statement
- Problem statement

# Design
-  For Bug, Describe the fix here.
-  For Feature, Post the link for design

# Coding
   Checklist for Author
-  [ ] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
-  [ ] New/Affected tests are executed on Latest Build
-  [ ] Attach test execution logs
-  [ ] Collection tested and no collection error introduced (`pytest --local True --collect-only`)

# Review Checklist 
  Checklist for Author
- [ ] JIRA number/GitHub Issue added to PR
- [ ] PR is self reviewed
- [ ] Jira and state/status is updated and JIRA is updated with PR link
- [ ] Check if the description is clear and explained

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
-  [ ] If change in any common function, make sure to update all calls and execute all affected tests.

# Documentation
  Checklist for Author
-  [ ] Changes done to ReadMe / WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

**Review by kulrk — APPROVED** [review]

File: tests/iostability/test_iostability_degraded_path.py [hunk 1]
```diff
@@ -94,6 +95,21 @@ def setup_class(cls):
...
+        sts = secrets.SystemRandom().sample(sts_list, 1)[0]
+        cls.delete_pod = sts_dict[sts][-1]
+        cls.log.info("Pod to be deleted is %s", cls.delete_pod)
+        cls.set_type, cls.set_name = cls.master_node_list[0].get_set_type_name(
+            pod_name=cls.delete_pod)
+        if cls.set_type == STATEFULSET:
+            resp = cls.master_node_list[0].get_num_replicas(cls.set_type, cls.set_name)
+            assert_utils.assert_true(resp[0], resp)
+            cls.num_replica = int(resp[1])
+        else:
```
Comment (sonalk0209) [inline tests/iostability/test_iostability_degraded_path.py:111]: if only for server and data pod, we may no longer need this else as going forward its going to be statefulset only.
Agree? @ApurwaMohite

Comment (ApurwaMohite) [reply tests/iostability/test_iostability_degraded_path.py:111]: If going forward data and server will be only statefulset then we don't need this if-else. We can by default assume that set_type is statefulset.

**Review by ApurwaMohite — APPROVED** [review]

File: tests/iostability/test_iostability_degraded_path.py (hunk 1 above)
Comment (sonalk0209) [reply tests/iostability/test_iostability_degraded_path.py:111]: @priyankaborawake  please change accordingly?

**Comment by priyankaborawake** [comment]
[collections.log](https://github.com/Seagate/cortx-test/files/9298056/collections.log)

**Review by ravindrachoudhari — APPROVED** [review]

**Review by sonalk0209 — APPROVED** [review]

**Pull request merged**